.craft.yml

```diff
@@ -1,7 +1,5 @@
 minVersion: 0.23.1
```

Comment:
this probably has to changed after released right

Reply:
Yes, but the feature hasn't been released yet on Craft. Once it's released, I'll get this branch ready (including updating the minimum Craft version).